components/director/pkg/resource/resource.go

```diff
@@ -27,6 +27,7 @@ const (
 	Webhook                    Type = "webhook"
 	Tenant                     Type = "tenant"
 	TenantIndex                Type = "tenantIndex"
+	Schema                     Type = "schema_migration"
```

Comment:
The other values are camel-cased, maybe `schemaMigration` for consistency? It should be safe, because it looks like the value is used for this log only - https://github.com/kyma-incubator/compass/blob/readiness-probes/components/director/internal/repo/get.go#L78